README.md

```diff
@@ -41,8 +41,8 @@ Check the downloads page on [podman-desktop.io/downloads](https://podman-desktop
 
 - [Podman container engine](https://github.com/containers/podman)
 - [crc](https://github.com/code-ready/crc)
-- [Podman Lima machines](https://github.com/lima-vm/lima)
-- [Docker](https://github.com/moby/moby)
+- [Lima virtual machines](https://github.com/lima-vm/lima)
+- [Docker container engine](https://github.com/moby/moby)
```

Comment:
```suggestion
- [Moby container engine](https://github.com/moby/moby)
```

maybe name it moby ?

Reply:
Or use https://github.com/docker/docker?

The binary is still called "dockerd"

Reply:
up to you, I would say name and link should be referencing the same so either moby/moby or docker/docker

Reply:
It does redirect anyway, but could be changed if you want it to match (I did not actually modify this part)